buildlibxml.py

```diff
@@ -38,7 +38,12 @@ def download_and_extract_windows_binaries(destdir):
         if release_path in filename
     ]
 
-    arch = "win64" if sys.maxsize > 2**32 else "win32"
+    if platform.machine() == 'ARM64':
+        arch = "win-arm64"
+    elif sys.maxsize > 2**32:
+        arch = "win64"
+    else:
+        arch = "win32"
     if sys.version_info < (3, 5):
         arch = 'vs2008.' + arch
```

Comment:
The last two lines should better go before the others now.
```suggestion
    if sys.version_info < (3, 5):
        arch = 'vs2008.' + arch
    elif platform.machine() == 'ARM64':
        arch = "win-arm64"
    elif sys.maxsize > 2**32:
        arch = "win64"
    else:
        arch = "win32"
```